rector.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+use RectorLaravel\Rector\MethodCall\RedirectRouteToToRouteHelperRector;
+use Rector\Set\ValueObject\LevelSetList;
+use Rector\Set\ValueObject\SetList;
+use RectorLaravel\Set\LaravelSetList;
+use Rector\PHPUnit\Set\PHPUnitSetList;
+
+
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->paths([
+        __DIR__ . '/app',
+        __DIR__ . '/config',
+        __DIR__ . '/database',
+        __DIR__ . '/public',
+        __DIR__ . '/resources',
+        __DIR__ . '/routes',
+        __DIR__ . '/rector.php',
+    ]);
+
+    $rectorConfig->sets([
+        SetList::DEAD_CODE,
+        SetList::CODE_QUALITY,
+        LevelSetList::UP_TO_PHP_82,
+        LaravelSetList::LARAVEL_100,
+        SetList::PHP_70,
+        SetList::PHP_71,
+        SetList::PHP_72,
+        SetList::PHP_73,
+        SetList::PHP_74,
+        SetList::PHP_80,
+        SetList::PHP_81,
+        SetList::PHP_82,
+        SetList::NAMING,
+        SetList::PHP_52,
+        SetList::PHP_53,
+        SetList::PHP_54,
+        SetList::PHP_55,
+        SetList::PHP_56,
+        SetList::PRIVATIZATION,
+        SetList::TYPE_DECLARATION,
+        SetList::EARLY_RETURN,
+        SetList::INSTANCEOF,
+        SetList::GMAGICK_TO_IMAGICK,
+        SetList::CODING_STYLE,
+        SetList::STRICT_BOOLEANS,
+        PHPUnitSetList::PHPUNIT_100,
+        PHPUnitSetList::PHPUNIT_CODE_QUALITY,
```

Comment:
Still Missing
---
PSR_4
LARAVEL_ARRAY_STR_FUNCTION_TO_STATIC_CALL
LARAVEL_CODE_QUALITY
LARAVEL_LEGACY_FACTORIES_TO_CLASSES
ARRAY_STR_FUNCTIONS_TO_STATIC_CALL

Not required
---
PHP_70,
PHP_71,
PHP_72,
PHP_73,
PHP_74,
PHP_80,
PHP_81,
PHP_52,
PHP_53,
PHP_54,
PHP_55,
PHP_56,

Duplicate
---
PHP_82,